Leader.js

```diff
@@ -0,0 +1,20 @@
+const SIZE_OF_ARRAY = +process.argv[2];
+const STRING_ARRAY = process.argv[3].trim().split(',')
+const NUMBER_ARRAY = STRING_ARRAY.map((ele) => { return +ele })
+const TEMP_ARRAY = [...NUMBER_ARRAY]; 
+
+let leaderList = [];
+
+function isLeader(number, arr) {
+   let greaterThanEle = arr.filter((numInArr) => {
+      return numInArr > number
+   })
+   if (greaterThanEle.length > 0) return false
+   else return true
+}
+
+NUMBER_ARRAY.forEach((ele) => {
+   TEMP_ARRAY.shift()
+   let leader = isLeader(ele, TEMP_ARRAY)
+   if (leader) leaderList.push(ele)
```

Comment:
can we keep it simple to iterate the reverse array then we can find the greater / leader values?

Reply:
Sorry didn't get concept of reversing the array.
As per my understating, we have to check that current number is greater than the right side element, even if we reverse the array then have to check for left hand side. 
Above I am sending each number along with right hand side number only as required.